client/components/payments-status/index.tsx

```diff
@@ -60,11 +60,12 @@ interface Props {
 const PaymentsStatus: React.FC< Props > = ( props ) => {
 	const { paymentsEnabled, accountStatus } = props;
 
-	if ( 'pending_verification' === accountStatus ) {
-		return <PaymentsStatusPending iconSize={ props.iconSize } />;
+	if ( paymentsEnabled ) {
```

Comment:
This change created a bit of confusion IMO
This is how it looks when Stripe returns status pending within first minute
![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/9771a284-ae19-44fb-af13-efd6423e9840)
And after making transaction and clearing the cache
![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/14206389-0bf6-4b2d-a37e-5955e446c774)

it used to be both payments and deposits`Pending verification`, now it's only deposits
WDYT?

Reply:
Thanks @oaratovskyi , great catch! 

So, the change to payments being `Enabled` is what I wanted 🙂  previously, it would show as `pending verification` in some cases which I think is misleading because the merchant may think payments are not enabled even though they are actually able to accept payments.

However, the deposits change from `pending verification` to `not connected` is not intended - I think it should stay `pending verification` in both cases. I'll take another look at that logic and see why that happens!

Reply:
> However, the deposits change from pending verification to not connected is not intended - I think it should stay pending verification in both cases. I'll take another look at that logic and see why that happens!

That's because status that Stripe returns changes from `pending_verification` to something else! Though it shows as Pedning on Stripe's side, I know it's confusing 😕

Reply:
Ah yes, you're right! I think I can fix it by adding a check if the user is in PO for the "disabled" deposits status.

Reply:
This should be addressed by the latest two commits: https://github.com/Automattic/woocommerce-payments/pull/7665/files#diff-6de3ec2c3cd386ba46aca51c5d7df994a402e52fea7b3ffcbcc1c1d6d887f2fe

Could you take another look and make sure it looks okay to you? Thanks! 🙂

Reply:
Now it works precisely with testing instructions 🚀 Thanks for addressing that!